canonical-web-and-design/cn.ubuntu.com
# add dynamic tablet page and data

### Done

Made the /tablet page use the CMS
### QA

`make import-new-pages`
`make run`
Navigate to /tablet and see that the correct content is there
### Card

https://canonical.leankit.com/Boards/View/111185042/118512991
